status/create_status.py

```diff
@@ -5,25 +5,37 @@
     log_add,
     log_exception,
 )
+from okdata.resource_auth import ResourceAuthorizer
 
 from status.StatusData import StatusData
-from status.common import response, response_error, is_owner
+from status.common import (
+    response,
+    response_error,
+    extract_dataset_id,
+    extract_bearer_token,
+)
 
 patch_all()
+resource_authorizer = ResourceAuthorizer()
 
 
 @logging_wrapper
 @xray_recorder.capture("create_status")
 def handler(event, context):
     db = StatusData()
-    item = simplejson.loads(event["body"])
+    status_item = simplejson.loads(event["body"])
     try:
-        caller_is_owner = is_owner(event, item)
-        log_add(is_owner=caller_is_owner)
-        if not caller_is_owner:
+        bearer_token = extract_bearer_token(event)
+        dataset_id = extract_dataset_id(status_item)
```

Comment:
Hva om denne returnerer `None`? Bør man få en 404 i stedet for at det sjekkes permissions for `okdata:dataset:None` nedenfor? (Samme spørsmål gjelder også de andre handlerene.)

Reply:
Da ville man blitt autorisert mot ressursen `"okdata:dataset:None"` og det vil vi ikke ha noe av

Reply:
Hm, det skjønte jeg ikke. Hvorfor blir man autorisert hvis man gir en 404? Slik det er nå vil jo permissions sjekkes mot `okdata:dataset:None` hvis `extract_dataset_id` returnerer `None`?

Reply:
Tenker at det burde bli returnert en 400 egentlig hvis dataset_id blir None her, siden det vil være en feil i request-bodyen